static/app/views/issueList/searchBar.tsx

```diff
@@ -53,6 +62,11 @@ const getFilterKeySections = (tags: TagCollection): FilterKeySection[] => {
       label: t('Event Tags'),
       children: eventTags,
     },
+    {
+      value: FieldKind.FEATURE_FLAG,
+      label: t('Event Feature Flags'),
```

Comment:
This is very long and causes this to wrap to 2 lines. We want to keep this at one line, so can we just call this "Flags"?